defaultconfig/template.conf

```diff
@@ -0,0 +1,196 @@
+# TODO: Convert from nginx conf comments to go templating comments
+# Number of worker processes running in container
+worker_processes 1;
+
+# Run NGINX in foreground (necessary for containerized NGINX)
+daemon off;
+
+# Set the location of the server's PID file
+pid {{ env "APP_ROOT" }}/logs/nginx.pid;
+
+# Set the location of the server's error log
+error_log {{ env "APP_ROOT" }}/logs/error.log;
+
+
+events {
+  # Set number of simultaneous connections each worker process can serve
+  worker_connections 1024;
+}
+
+http {
+ # consider adjusting the buffer size (for POST requests) to best serve the
+ # Dockerized context?
+ # client_body_buffer_size 10K;
+ # client_header_buffer_size 1k;
+ # client_max_body_size 8m;
+ # large_client_header_buffers 2 1k;
+
+# TODO: Can we write these files to /tmp instead?
+  client_body_temp_path {{ env "APP_ROOT" }}/client_body_temp;
+  proxy_temp_path {{ env "APP_ROOT" }}/proxy_temp;
+  fastcgi_temp_path {{ env "APP_ROOT" }}/fastcgi_temp;
+
+# TODO: Why set this?
+  charset utf-8;
```

Comment:
This sets the `Content-Type` response header correctly and does some simple conversion for files that are not `utf-8`. Its the most widely used character set and so makes sense for us to default to instead of saying nothing which could be more problematic for some clients.